### ISSUE TITLE
Does the code run?

### ISSUE THREAD

**Issue by Ferassaad** [ferdialaca/Carla_Detection_YOLOv5#1]
did this project work? if so what version python did you use for this? and if you could update readme file with more information.

**Comment by ferdialaca** [comment]
Yes it works. We use Python 3.8.8. You can find more detail from https://www.linkedin.com/feed/update/urn:li:activity:7089909034249519104/

**Comment by Ferassaad** [comment]
How can i get in contact with you?
